app/presenters/auction_status_presenter_factory.rb

```diff
@@ -55,10 +72,86 @@ def end_date
   end
 
   def sign_in_link
-    link_to 'Sign in', sign_in_path
+    Url.new(link_text: 'Sign in', path_name: 'sign_in')
   end
 
   def sign_up_link
-    link_to 'sign up', sign_up_path
+    Url.new(link_text: 'sign up', path_name: 'sign_up')
+  end
+end
+
+class FutureUserIsGuest
+  attr_reader :auction
+
+  def initialize(auction:)
+    @auction = auction
+  end
+
+  def to_s
+    "This auction starts on #{start_date}. #{sign_in_link} or
+    #{sign_up_link} with GitHub to bid."
+  end
+
+  private
+
+  def start_date
+    DcTimePresenter.convert_and_format(auction.started_at)
+  end
+
+  def sign_in_link
+    Url.new(link_text: 'Sign in', path_name: 'sign_in')
+  end
+
+  def sign_up_link
+    Url.new(link_text: 'sign up', path_name: 'sign_up')
+  end
+end
+
+class FutureUserIsVendor
```

Comment:
(I would opt to do that update separately since in this PR we are just extending an existing pattern)